packages/frontend-core/src/api/app.ts

```diff
@@ -136,6 +136,14 @@ export const buildAppEndpoints = (API: BaseAPIClient): AppEndpoints => ({
    * @param app the app to create
    */
   createApp: async app => {
+    if (app instanceof FormData) {
+      return await API.post({
+        url: "/api/applications",
+        body: app,
+        json: false,
```

Comment:
Instead of doing this `instanceof` check, you could simply pass the **json** boolean as an optional flag that defaults to true